packages/extester/src/util/codeUtil.ts

```diff
@@ -55,19 +57,27 @@ export class CodeUtil {
     private cliEnv!: string;
     private availableVersions: string[];
     private extensionsFolder: string | undefined;
+    private extensionDevelopmentPath: string | undefined;
+    private coverage: boolean | undefined;
 
     /**
      * Create an instance of code handler
      * @param folder Path to folder where all the artifacts will be stored.
      * @param extensionsFolder Path to use as extensions directory by VS Code
      */
-    constructor(folder: string = DEFAULT_STORAGE_FOLDER, type: ReleaseQuality = ReleaseQuality.Stable, extensionsFolder?: string) {
+    constructor(folder: string = DEFAULT_STORAGE_FOLDER, type: ReleaseQuality = ReleaseQuality.Stable, extensionsFolder?: string, coverage?: boolean) {
         this.availableVersions = [];
         this.downloadPlatform = this.getPlatform();
         this.downloadFolder = path.resolve(folder);
         this.extensionsFolder = extensionsFolder ? path.resolve(extensionsFolder) : undefined;
+        this.coverage = coverage;
         this.releaseType = type;
 
+        // If code coverage is enabled, set extensionDevelopmentPath to the current directory.
+        if (this.coverage) {
+            this.extensionDevelopmentPath = path.resolve(".");
```

Comment:
can we use directly `process.cwd()` here?

Reply:
I will try that and update the result here. My current understanding is that we still need to use the `extensionDevelopmentPath` parameter internally for enabling code coverage even though the ExTester CLI will not support it as its own start-up paramter.

Reply:
Inserted `process.cwd()` in `browser.ts` and removed these three lines in the latest commit.